jena-text/src/test/java/org/apache/jena/query/text/assembler/TestTextIndexLuceneAssembler.java

```diff
@@ -97,6 +97,14 @@
             index.close();
         }
     }
+    
+    @Test public void testOneIndexPerDirectoryMem() {
```

Comment:
Should there be a corresponding test case that validates that when you have two different nodes specified as RAM indexes you get two different `TextIndexLucene` instances?